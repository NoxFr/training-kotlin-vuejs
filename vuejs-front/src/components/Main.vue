<template>
    <v-container grid-list-xl>
        <tkv-search-form></tkv-search-form>
        <tkv-beer-list></tkv-beer-list>
    </v-container>
</template>

<script>
import TkvSearchForm from '@/components/SearchForm'
import TkvBeerList from '@/components/BeerList'

export default {
  name: 'tkvMain',
  components: {
    TkvSearchForm,
    TkvBeerList
  }
}
</script>
