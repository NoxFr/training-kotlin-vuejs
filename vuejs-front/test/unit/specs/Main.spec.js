import { shallow } from '@vue/test-utils'
import Main from '@/components/Main'
import Vue from 'vue'
import Vuetify from 'vuetify';
Vue.use(Vuetify);

describe('Main.vue', () => {
  it('should match snapshot', () => {
    // when
    const wrapper = shallow(Main)

    // then
    expect(wrapper.element).toBeDefined();
    expect(wrapper.element).toMatchSnapshot();
  })
})
