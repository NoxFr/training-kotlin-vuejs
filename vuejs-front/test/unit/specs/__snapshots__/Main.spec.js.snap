// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main.vue should match snapshot 1`] = `<!---->`;
